workflow/scripts/krakenuniq2krona.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+from sys import argv
+import pandas as pd
+
+krakenuniq_output = argv[1]
+krakenuniq_seqs = argv[2]
+
+n_unique_kmers = int(argv[3])
+n_tax_reads = int(argv[4])
+
+# Read and filter KrakenUniq output
+kraken_output_df = pd.read_csv(krakenuniq_output, delimiter="\t", comment="#")
+print(f"Original data set dimensions: {kraken_output_df.shape}")
+
+kraken_output_df = kraken_output_df[kraken_output_df["kmers"] > n_unique_kmers]
+
+print(f"Data set dimensions after breadth of coverage filter: {kraken_output_df.shape}")
+
+kraken_output_df = kraken_output_df[kraken_output_df["taxReads"] > n_tax_reads]
+kraken_output_df = kraken_output_df[kraken_output_df["rank"] == "species"]
+kraken_output_df.rename(columns={"%": "Pers_Reads"}, inplace=True)
```

Comment:
Percent_Reads? This is percentage reads no?

Reply:
I had left the "%" column name at some stage, but I wasn't sure it would break something, so I reverted to the previous. I can try and investigate

I have tested this on a large-ish example and the outputs were near-identical, save for some lines that were swapped (because of different sorting strategies between python and R)

Reply:
This column name is used in other scripts too: 

```
scripts/filter_krakenuniq.R:colnames(df)[1] <- "Pers_Reads"
scripts/krakenuniq2krona.py:kraken_output_df.rename(columns={"%": "Pers_Reads"}, inplace=True)
scripts/krakenuniq2krona.py:kraken_output_df = kraken_output_df.sort_values(by="Pers_Reads", ascending=False)
scripts/krakenuniq_abundance_matrix.R: names(df[[i]])[1]<-"Pers_Reads"
```

We should probably change this in a separate PR

Reply:
Ah, I see. Yes, that can wait, not a big deal.